src/types.ts

```diff
@@ -41,6 +45,13 @@ export interface OverrideObject {
 
 export type OverrideStandard = StandardType & OverrideObject;
 
+export type OutputType = 'json' | 'yaml';
```

Comment:
And last suggestion 😅  `yaml` can be also named as `yml` so we should also support it. Having `toProperFormat` function you need to make only one addition check in `if` statement :)